### PR TITLE
PTTB-14 #comment Added eye-icon and its functionality on password field

### PR DESCRIPTION
**Pull request by NirmalShrestha01**
Added eye-icon and its functionality on the password field, 
please review once.

### CODE REVIEW

**Comment by alex1the1great** [comment]
@NirmalShrestha01, the code seems fine to me, but if want to implement this feature in different places like the sign-up page.
What about creating a reusable component?

**Comment by NirmalShrestha01** [comment]
> 

Sure Asim will work on it. Thank you!

File: src/modules/accounts/views/ViewLogin.vue [hunk 1]
```diff
@@ -114,14 +119,19 @@ async function handleLogin() {
           label="Email address"
           v-model="form.email"
           :error="emailError"
+          data-cy="your email address"
```
Comment (alex1the1great) [inline src/modules/accounts/views/ViewLogin.vue:122]: data-cy="emailAddress"

File: src/components/PasswordToggle.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { ref } from "vue";
+
+let showPassword = ref(false);
```
Comment (alex1the1great) [inline src/components/PasswordToggle.vue:4]: Remove the "showPassword" state, and add this to the parent component.

File: src/components/PasswordToggle.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+</script>
+
+<template>
+  <span class="toggle_button">
+    <i class="bi toggle_icon" :class="showPassword ? 'bi-eye' : 'bi-eye-slash'" id="showPassword"
+      @click="showPassword = !showPassword"></i>
```
Comment (alex1the1great) [inline src/components/PasswordToggle.vue:10]: When the icon is clicked, emit an event which will trigger the parent component to change the state.

File: src/modules/accounts/views/ViewLogin.vue [hunk 4]
```diff
@@ -114,14 +119,19 @@ async function handleLogin() {
...
+        <div class="input_form">
+          <BaseInput
+            :type="showPassword ? 'text' : 'password'"
+            name="password"
+            label="Password"
+            v-model="form.password"
+            :error="passwordError"
+          />
+
+          <PasswordToggle @click="showPassword = !showPassword" />
```
Comment (alex1the1great) [inline src/modules/accounts/views/ViewLogin.vue:133]: Replace the "@click" with the event emitted from the child component and change the state.

**Review by alex1the1great — APPROVED** [review]

**Pull request merged**